room-battle.js

```diff
@@ -232,8 +241,11 @@ class BattleTimer {
 					perTurnTicks = 0;
 				}
 			}
-			this.ticksLeft[slotNum] += this.settings.perTurnTicks;
```

Comment:
Also: Apparently my `perTurnTicks` changes did nothing! We're still doing `+= this.settings.perTurnTicks` instead of `+= perTurnTIcks`. That'll need to be fixed.

Reply:
I fixed it; you'll need to rebase.